themes/foulab-2018/layouts/partials/header.html

```diff
@@ -1,39 +1,41 @@
-      <div class="header">
-	<canvas id="gl_canvas"></canvas>
+<div id="header">
+  <canvas id="gl_canvas"></canvas>
 
-	<div id="status">
-		STATUS
-		<span style="font-size:8px;font-family:monospace;white-space:pre;">
+  <div id="status">
+    STATUS
+    <span style="font-size:8px;font-family:monospace;white-space:pre;">
  ██████  ██████  ███████ ███    ██
 ██    ██ ██   ██ ██      ████   ██
 ██    ██ ██████  █████   ██ ██  ██
 ██    ██ ██      ██      ██  ██ ██
  ██████  ██      ███████ ██   ████
-			</span>
-	</div>
+    </span>
+  </div>
 
-	<div class="tinybar">
-	  
-	    <a href="https://github.com/FOULAB" title="Foulab GitHub">
-	      <img id="icons" src="{{ .Site.BaseURL }}/img/git.png"/>
-	    </a>
-	    <a href="https://web.libera.chat/?channels=#foulab" title="IRC Webchat">
-	      <img id="icons" src="{{ .Site.BaseURL }}/img/irc.png"/>
-	    </a>
-	    <a href="https://twitter.com/foulab" title="Twitter">
-	      <img id="icons" src="{{ .Site.BaseURL }}/img/twt.png"/>
-	    </a>
-	    <a href="{{ .Site.BaseURL }}/news/index.xml" title="RSS Feed">
-	      <img id="icons" src="{{ .Site.BaseURL }}/img/rss.png"/>
-	    </a>
-	    <a href="mailto:friends.of.foulab-subscribe@lists.riseup.net" title="Mailing List">
-	      <img id="icons" src="{{ .Site.BaseURL }}/img/eml.png"/>
-	    </a>
-	    <a href="https://www.facebook.com/groups/144161830009" title="Facebook Group">
-	      <img id="icons" src="{{ .Site.BaseURL }}/img/fb.png"/>
-	    </a>
-	  
-	</div>
-	
-      </div>
-      
+  <div class="tinybar">
+
+    <a href="https://github.com/FOULAB" title="Foulab GitHub">
+      <img class="icons" src="/img/git.png" alt="Link to Foulab GitHub">
+    </a>
+    <a href="https://test.foulab.org/" title="Mattermost Server">
```

Comment:
I'm on the fence about linking to Mattermost.
- for the majority of visitors to the site (non-members, people who have never been to the space), they won't be able to log in, so it won't be useful and would probably just raise questions
- Mattermost is a bit of an 'open secret' for people who have visited the lab in person, and I think that's a nice thing.

@kryma-foulab @0x5c what do you think?

Reply:
Yeah I wasn't sure about this one either - I can easily revert this if the consensus is that it shouldn't be there. Personally I had no idea what the URL was and needed to login from my phone at one point, so this would have helped me... but it's probably a rare occurrence.

Reply:
Probably not a good idea to put Mattermost - although it is a visual treat to see the thing in the sidebar :)
If you'd like to get fancy with it, is there any good way to serve it only to the lab's IP?

Reply:
Hiding it visually might be possible but this would be client-side so it would still be viewable by someone looking at the source code, so not really a good solution.

Reply:
Hiding Mattermost does not need to be 'secure', in my opinion, (heck it might even be a fun game that you get access if you view the source code and solve a puzzle ;-) ) but I just think it would be brittle and tedious to maintain an IP allowlist. So I suggest simply leave it out.